backend/src/modules/transactions/dto/transaction.dto.ts

```diff
@@ -21,7 +21,7 @@ export class TransactionDTO {
 
   @ApiModelProperty()
   @IsString()
-  readonly Category: string;
+  Category: any;
```

Comment:
manau reiketu palikti kazkoki type. Jei cia laikomas kategorijos `id` tai gali likti `string`, nes schemoje jis bus `string`. O jei ne `string`, tai ka ten virsuj veikia `@IsString()`? :DD

Reply:
man atrodo @klaidas01 planavo embeddint tas kategorijas į transaction dto objektą, bet man kažkaip atrodo, kad mes galim biškį kitaip šitą padaryt, gal daugiau papasakosiu per standup'ą mūsų